webrtc-kmp/src/jvmMain/kotlin/com/shepeliev/webrtckmp/LocalAudioStreamTrack.kt

```diff
@@ -0,0 +1,15 @@
+package com.shepeliev.webrtckmp
+
+import dev.onvoid.webrtc.media.audio.AudioTrack
+import dev.onvoid.webrtc.media.audio.AudioTrackSource
+
+internal class LocalAudioStreamTrack(
+    native: AudioTrack,
+    private val audioSource: AudioTrackSource,
+    override val constraints: MediaTrackConstraints,
+) : MediaStreamTrackImpl(native), AudioStreamTrack {
+
+    override fun onStop() {
+        // audioSource.dispose()
```

Comment:
I think we shouldn't leave commented code here. If JVM `AudioTrackSource` doesn't implement `dispose` we merely don't override `onStop` here. What would you say?

Reply:
WIll do 👍